<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingParamDialog</class>
 <widget class="QDialog" name="SettingParamDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QSpinBox" name="spinBoxYLen">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxXLen">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>評価する領域のX方向サイズ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>評価する領域のX方向サイズ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>登録ライブラリファイル名</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditRegPackFileName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSelectFile">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>41</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>判定決定時の最低適合率</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxMinMatchRate">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
   <property name="minimum">
    <double>-1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QToolButton" name="toolButtonLayer0">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>LY0</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonLayer1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>LY1</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonLayer2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>LY2</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>使用するレイヤー</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
